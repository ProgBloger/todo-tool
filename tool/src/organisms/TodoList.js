import React from "react";
import TodoItem from "../molecules/TodoItem";

const TodoList = () => {

    const content = () => {
        return [{id:1},{id:2},{id:3},{id:4},{id:5}].map(item => <TodoItem key={item.id}/>)
    }
    
    console.log(content());

    return (
        <ul>
            {content()}
        </ul>
    )
}

export default TodoList;