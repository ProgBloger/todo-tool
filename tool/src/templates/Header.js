import bgImage from '../assets/Mountain.jpg';
import styles from './Header.module.scss';

const Header = () => {
    return (
        <div className={styles['main-image']}>
            <img src={bgImage} alt="Magenta Background Car" />
        </div>
    )
}

export default Header;