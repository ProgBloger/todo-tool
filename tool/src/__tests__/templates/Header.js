import renderer from 'react-test-renderer';
import expect from 'expect';
import TodoList from '../../organisms/TodoList';
import Header from '../../templates/Header';

it('renders', () => {
    // Arrange
    // Act
    var tree = renderer.create(
        <Header />
    ).toJSON();

    // Assert
    expect(tree).toMatchSnapshot();
})