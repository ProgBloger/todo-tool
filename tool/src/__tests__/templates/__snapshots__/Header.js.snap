// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="main-image"
>
  <img
    alt="Magenta Background Car"
    src="Mountain.jpg"
  />
</div>
`;
